tests/sample-v2-addon/tests/integration/components/ui/page-test.ts

```diff
@@ -8,18 +7,31 @@ import { setupRenderingTest } from '../../../helpers';
 module('Integration | Component | ui/page', function (hooks) {
   setupRenderingTest(hooks);
 
-  test('The component handles the page layout', async function (this: TestContext, assert) {
-    await render<TestContext>(hbs`
-      <Ui::Page
+  test('The component handles the page layout', async function (assert) {
+    await render(hbs`
+      <Ui::PageNew
         @title="Form"
       >
         <div data-test-body>
         </div>
-      </Ui::Page>
+      </Ui::PageNew>
     `);
 
     assert.dom('h1').hasText('Form', 'We see the title.');
 
     assert.dom('[data-test-body]').exists('We see the yielded content.');
   });
+
+  test('CSS modules', async function (assert) {
+    await render(hbs`
+      <Ui::PageNew
+        @title="Form"
+      >
+        <div data-test-body>
+        </div>
+      </Ui::PageNew>
+    `);
+
+    assert.dom('h1').hasClass(/^sample-v2-addon/, 'We see the local style.');
```

Comment:
A new test for `<Ui::Page>` component. The test asserts that CSS modules can be implemented in v2 addons.